CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.12) # Don't bump this version for no reason
+cmake_minimum_required(VERSION 3.25) # Don't bump this version for no reason
```

Comment:
Let's drop the CMake changes - v3.25 is not easily accessible yet

Reply:
Should we also drop BLA_SIZEOF_INTEGER?
So that current version has no need to change

Reply:
Ah, let me add some if condition on the cmake version